index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name ="viewport" content="width= device-width, initial-scale = 1.0">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="styles.css">
+        <title>Munana Clinton </title>
+     </head>
+     <body>
+        <header id="header">
+            <nav class="header-nav">
+                <h3 id="logo">Munana Clinton</h3>
```

Comment:
- [ ] Kindly make sure to change the content of this `h3` to a link (`a`) because three reasons: convention, semantics, and user experience(UX)  [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info